.changelogs/course-builder-editor.yml

```
@@ -0,0 +1,3 @@
+significance: minor
```

```suggestion
significance: patch
```
I think the significance of this is rather `patch` here you know, not a functional  addition.

---

Back to minor because probably it's still an addition, sorry Lee.